src/pcp/types.rs

```diff
@@ -112,6 +128,8 @@ impl<F: FieldElement> PolyCheckedVector<F> {
 }
 
 impl<F: FieldElement> Value<F, PolyEval<F>> for PolyCheckedVector<F> {
+    type Param = Vec<F>; // A polynomial
```

Comment:
Ah, wouldn't it be nice if we had a `Polynomial` type so we wouldn't have to use comments to indicate when a `[F]` or `Vec<F>` is actually a polynomial 😄

Reply:
Hahaha. Yeah.

To make this explicit, we could do something like
```rust
type Param = PolyCheckedVector<F>;
```
where
```rust
pub struct PolyCheckedVectorParam<F: FieldElement> {
   pub poly: Vec<F>,
}
```
I just figure this was overkill here. (Though I'm not sure if the associated type declaration would need a type parameter?)